documentation/en/cli-lotus-miner.md

```diff
@@ -165,173 +175,19 @@ USAGE:
-
-OPTIONS:
-   --offline   create backup without the node running (default: false)
-   --help, -h  show help
-```
-
-## lotus-miner auth
-```
-NAME:
-   lotus-miner auth - Manage RPC permissions
-
-USAGE:
-   lotus-miner auth command [command options] [arguments...]
-
-COMMANDS:
-   create-token  Create token
-   api-info      Get token with API info required to connect to this node
-   help, h       Shows a list of commands or help for one command
-
-OPTIONS:
-   --help, -h  show help
-```
-
-### lotus-miner auth create-token
-```
-NAME:
-   lotus-miner auth create-token - Create token
-
-USAGE:
-   lotus-miner auth create-token [command options] [arguments...]
-
-OPTIONS:
-   --perm value  permission to assign to the token, one of: read, write, sign, admin
-   --help, -h    show help
-```
-
-### lotus-miner auth api-info
-```
-NAME:
-   lotus-miner auth api-info - Get token with API info required to connect to this node
-
-USAGE:
-   lotus-miner auth api-info [command options] [arguments...]
-
-OPTIONS:
-   --perm value  permission to assign to the token, one of: read, write, sign, admin
-   --help, -h    show help
-```
-
-## lotus-miner log
-```
-NAME:
-   lotus-miner log - Manage logging
-
-USAGE:
-   lotus-miner log command [command options] [arguments...]
-
-COMMANDS:
-   list       List log systems
-   set-level  Set log level
-   alerts     Get alert states
-   help, h    Shows a list of commands or help for one command
-
-OPTIONS:
-   --help, -h  show help
-```
-
-### lotus-miner log list
-```
-NAME:
-   lotus-miner log list - List log systems
-
-USAGE:
-   lotus-miner log list [command options] [arguments...]
-
-OPTIONS:
-   --help, -h  show help
-```
-
-### lotus-miner log set-level
-```
-NAME:
-   lotus-miner log set-level - Set log level
-
-USAGE:
-   lotus-miner log set-level [command options] [level]
-
-DESCRIPTION:
-   Set the log level for logging systems:
-
-      The system flag can be specified multiple times.
-
-      eg) log set-level --system chain --system chainxchg debug
-
-      Available Levels:
-      debug
-      info
-      warn
-      error
-
-      Environment Variables:
-      GOLOG_LOG_LEVEL - Default log level for all log systems
-      GOLOG_LOG_FMT   - Change output log format (json, nocolor)
-      GOLOG_FILE      - Write logs to file
-      GOLOG_OUTPUT    - Specify whether to output to file, stderr, stdout or a combination, i.e. file+stderr
-
+Online backups:
+For security reasons, the daemon must be have LOTUS_BACKUP_BASE_PATH env var set
+to a path where backup files are supposed to be saved, and the path specified in
+this command must be within this base path
 
 OPTIONS:
-   --system value [ --system value ]  limit to log system
-   --help, -h                         show help
-```
+   --offline                      create backup without the node running (default: "false")
+   --help, -h                     show help (default: "false")
 
-### lotus-miner log alerts
```

Comment:
why did this one come out of order? do you know why urfave puts it here but yours doesn't?

Reply:
Oh, I get why this is shuffled now -- the original is following the order as they appear in the `COMMANDS:` output, but yours is following the naive order they appear in the `Commands` slice. I think we should follow that same order—either retain it when you print `COMMANDS:` or just follow this algorithm:

1. Iterate over the `Commands` slice and print out any that _don't_ have a `Category`
2. Iterate over the `Commands` slice again and print out any that _do_ have a `Category`

I don't know if there's a sorting on `Category` name applied, maybe it won't matter for now.